external/common_functions.sh

```diff
@@ -219,6 +219,21 @@ function set_test_info() {
         ubi_packages="git wget unzip"
         ubi_minimal_packages="${ubi_packages}"
         ;;
+    external_custom)
+        github_url="https://github.com/apache/flink.git"
```

Comment:
Instead of "https://github.com/apache/flink.git" we will eventually use a variable so that we can pass in various URLs to application repositories

Reply:
Am waiting for the variable on this.

Reply:
where should I save the vaeriable?
I need more light on this 
Thanks

Reply:
If you have tried running this locally, you will have seen the 'flow' that is used.  

`make compile` invokes the build.xml ant script 
`make _external_custom` calls external.sh (which subsequently invokes the other shell scripts including common_functions.sh)

as a trial, you could see if you could set a new environment variable called CUSTOM_GITHUB_URL set with the value of "https://github.com/apache/flink.git" and use that environment variable in the common_functions.sh script.  

Alternatively, you can try passing in a variable to the script by the same name, but for that, you will have to understand the flow of the commands and scripts and understand what pieces call what other pieces.

Reply:
The update is done.
Waiting for review.